test/ndb_api_helpers_test.go

```diff
@@ -604,3 +605,50 @@ func TestGenerateProvisioningRequestReturnsErrorIfSSHKeyIsEmpty(t *testing.T) {
 		}
 	}
 }
+
+func TestGetActionArgumentsByDatabaseType(t *testing.T) {
+	// Test with MySQL database type
+	mysqlArgs, err := v1alpha1.GetActionArgumentsByDatabaseType(v1alpha1.DATABASE_TYPE_MYSQL)
+	if mysqlArgs != nil {
+		t.Error("Expected non-nil, but got nil")
```

Comment:
The error message and the if-condition look off. Can you double check? I believe the same is true in couple of more places in the file.